### ISSUE TITLE
Tutorials, content and documentation roadmap

### ISSUE THREAD

**Issue by schae234** [UMN-EGGL/HorseGeneAnnotation#3]

**Comment by MollyMcCue-DVM-PhD** [comment]
One thing that I think we need to make easy is getting right to where users can edit the tutorial documents (i.e. a direct link, and editing in a familiar format). Getting users from the equine community like Mickelson to contribute to those docs via this GitHub platform or using MD is an unreasonable expectation in many ways due to the learning curve to get to where they can edit the documents. We need to come up with a way to simplify this most scientists are not going to see the value in learning GitHub just to contribute to this project--its too much work

**Comment by schae234** [comment]
Yes, this is possible but integrating something so that edits are automatically incorporated via Word Docs or another format will be extra overhead. There are word [plugins](www.writage.com/) that will convert between MD and doc files. You can also drag and drop word docs into github comments, which we might be able to leverage as a submission system.

But there has to be a middle ground somewhere? If we want true crowd-contributions, we need to be able to handle a crowd. Handling the one-off contribution thats outside of the guidelines is fine, but my worry is that if there are dozens of emails and docs flying around it gets unwieldy very quickly.

We can also make some more upfront guides (videos, step-by-steps, etc) on how to contribute through github and put those right at the top of all the "non-technical" sections.
